src/app/services/bucket/baseBucketService.py

```diff
@@ -0,0 +1,21 @@
+from abc import ABC
+from abc import abstractmethod
+
+
+class BaseBucketService(ABC):
+    """Abstract BucketService class to setup an interface regardless of the actual service."""
+
+    def __init__(self):
+        super().__init__()
```

Comment:
I think you don't need to override this considering you're using it exactly as the inherited constructor!